templates/puppetserver-ingress-compilers.yaml

```diff
@@ -1,8 +1,10 @@
-{{- if .Values.puppetserver.compilers.enabled }}
-{{- if .Values.puppetserver.compilers.ingress.enabled -}}
+{{- if and .Values.puppetserver.compilers.enabled .Values.puppetserver.compilers.ingress.enabled -}}
 {{- $releaseName := .Release.Name -}}
 {{- $serviceName := include "puppetserver.puppetserver-compilers.serviceName" . }}
 {{- $servicePort := .Values.puppetserver.compilers.service.ports.puppetserver.port -}}
+{{- $pathType := .Values.puppetserver.compilers.ingress.pathType | default "ImplementationSpecific" -}}
+{{- $apiIsStable := eq (include "puppetserver.ingress.isStable" .) "true" -}}
+{{- $ingressSupportsPathType := eq (include "puppetserver.ingress.supportsPathType" .) "true" -}}
 apiVersion: networking.k8s.io/v1
```

Comment:
Shouldn't this use `puppetserver.ingress.apiVersion`?

Reply:
Not exactly - but tweaked it further. Thank you!